### ISSUE TITLE
Searching for Steam.ico

### ISSUE THREAD

**Issue by bluuman** [Marc3842h/Titan#64]
## Description

I am on Ubuntu 17.04, searching for steam.ico fails at /home/$USER/Resources. I double checked and I am using the binary from the tar.gz archive.

## Reproduction Steps

1.Launch exe with mono

## Errors

```bash
21:21:27 [Main] INFORMATION Quartz.NET Scheduler - Quartz scheduler 'DefaultQuartzScheduler' initialized
21:21:27 [Main] INFORMATION Quartz.NET Scheduler - Quartz scheduler version: 2.5.0.0
21:21:27 [Main] INFORMATION Quartz.NET Scheduler - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED started.
21:21:27 [Main] DEBUG Titan - Startup: Refreshing Steam Universe list.
21:21:27 [Main] DEBUG Quartz.NET Scheduler - Batch acquisition of 0 triggers
21:21:27 [Main] DEBUG Titan - Startup: Parsing Command Line Arguments.
21:21:27 [Main] INFORMATION Titan - The arguments --target and --mode were omitted - opening the UI.
21:21:27 [Main] DEBUG Titan - Startup: Loading UI Manager, Victim Tracker, Account Manager and Ban Manager.

Unhandled Exception:
System.IO.FileNotFoundException: Could not find file "/home/jalius/Resources/Steam.ico"
File name: '/home/jalius/Resources/Steam.ico'
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.Boolean anonymous, System.IO.FileOptions options) [0x0025f] in <8f2c484307284b51944a1a13a14c0266>:0 
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share) [0x00000] in <8f2c484307284b51944a1a13a14c0266>:0 
  at (wrapper remoting-invoke-with-check) System.IO.FileStream:.ctor (string,System.IO.FileMode,System.IO.FileAccess,System.IO.FileShare)
  at System.IO.File.Open (System.String path, System.IO.FileMode mode) [0x00000] in <8f2c484307284b51944a1a13a14c0266>:0 
  at Titan.UI.SharedResources..ctor () [0x00022] in <7288e9d7bcf0424fb96b10148781d945>:0 
  at Titan.UI.UIManager..ctor () [0x00027] in <7288e9d7bcf0424fb96b10148781d945>:0 
  at Titan.Titan.Main (System.String[] args) [0x00157] in <7288e9d7bcf0424fb96b10148781d945>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.IO.FileNotFoundException: Could not find file "/home/jalius/Resources/Steam.ico"
File name: '/home/jalius/Resources/Steam.ico'
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.Boolean anonymous, System.IO.FileOptions options) [0x0025f] in <8f2c484307284b51944a1a13a14c0266>:0 
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share) [0x00000] in <8f2c484307284b51944a1a13a14c0266>:0 
  at (wrapper remoting-invoke-with-check) System.IO.FileStream:.ctor (string,System.IO.FileMode,System.IO.FileAccess,System.IO.FileShare)
  at System.IO.File.Open (System.String path, System.IO.FileMode mode) [0x00000] in <8f2c484307284b51944a1a13a14c0266>:0 
  at Titan.UI.SharedResources..ctor () [0x00022] in <7288e9d7bcf0424fb96b10148781d945>:0 
  at Titan.UI.UIManager..ctor () [0x00027] in <7288e9d7bcf0424fb96b10148781d945>:0 
  at Titan.Titan.Main (System.String[] args) [0x00157] in <7288e9d7bcf0424fb96b10148781d945>:0 


```

## System Information

**Operating System**:Ubuntu 17.04
**Titan version**: 1.5.0 from tar.gz

**Comment by bluuman** [comment]
Nevermind, its my fault. You need to cd into the Titan folder before using it.

**Comment by Marc3842h** [comment]
I was aware of this and another person messaged me regarding this on Discord a few days ago. I implemented this now finally in be8883f2a134551ed26ed21041cfcfbdae8800e7.
